contributors/guide/get-started.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Getting Started"
+weight: 3
+slug: "get-started" 
```

Comment:
```suggestion
slug: "getting-started" 
```
The filename itself should also be renamed to match.
cc @parispittman
For thoughts on this.